mailbox/api/src/main/java/org/apache/james/mailbox/MailboxManager.java

```diff
@@ -440,7 +441,7 @@
      * @param mailboxAnnotations    the list of annotation should be insert/udpate/delete
      * @throws MailboxException in case of selected mailbox does not exist
      */
-    void updateAnnotations(MailboxPath mailboxPath, MailboxSession session, List<MailboxAnnotation> mailboxAnnotations) throws MailboxException;
+    void updateAnnotations(MailboxPath mailboxPath, MailboxSession session, List<MailboxAnnotation> mailboxAnnotations) throws MailboxException, AnnotationException;
```

Comment:
Why adding this exception to signature when its already covered y MailboxException ?